### ISSUE TITLE
windows.netscan.NetScan not working for Win10-x86

### ISSUE THREAD

**Issue by fgomulka** [volatilityfoundation/volatility3#532]
**Describe the bug**
There is an image of Windows10 which returns an error

**Context**
Volatility Version:  Volatility 3 Framework 1.0.1
Operating System:  Win10-x86
Python Version:  3.9.1
Suspected Operating System:  win10-x86
Command:  python3 vol.py --single-location win10-x86-2016-07-08.lime -q windows.netscan.NetScan

**To Reproduce**

1. Download the windows image win10-x86-2016-07-08.lime from https://buildbot.volatilityfoundation.org/volatility3/images/download/
1. Use command found above.
1. See error

**Expected behavior**
The plugin should not return this error 

**Screenshots**
<img width="828" alt="Screen Shot 2021-07-12 at 12 14 06 PM" src="https://user-images.githubusercontent.com/60993471/125321428-a9083e80-e30a-11eb-81ee-9badb253bcfe.png">

**Additional information**
N/A

**Comment by ikelos** [comment]
@japhlange This might be one for you?

**Comment by japhlange** [comment]
Sure. Sorry for hiding behind rocks, life and stuff.

I unfortunately cannot download the image and reproduce it :( Could you run `windows.info` on the image? 

In theory we have symbols for x86 10.0.10240 and 10.0.10586.

**Comment by fgomulka** [comment]
> ols for x86 10.0.10240 and 10.0.10586.

Here's the output of windows.info: 
```

Variable	Value

Kernel Base	0x81e1b000
DTB	0x1a8000
Symbols	file:///Users/fgomulka/vfoundation/volatility3/volatility3/symbols/windows/ntkrpamp.pdb/44B89EEAB81F466591D6570331648068-1.json.xz
Is64Bit	False
IsPAE	True
primary	0 WindowsIntelPAE
memory_layer	1 LimeLayer
base_layer	2 FileLayer
KdDebuggerDataBlock	0x8200f820
NTBuildLab	10586.420.x86fre.th2_release_sec
CSDVersion	0
KdVersionBlock	0x8200fef8
Major/Minor	15.10586
MachineType	332
KeNumberProcessors	2
SystemTime	2016-07-08 18:47:14
NtSystemRoot	C:\WINDOWS
NtProductType	NtProductWinNt
NtMajorVersion	10
NtMinorVersion	0
PE MajorOperatingSystemVersion	10
PE MinorOperatingSystemVersion	0
PE Machine	332
PE TimeDateStamp	Sat May 28 03:59:14 2016
```

**Comment by ikelos** [comment]
It looks like `AddressFamily` is part of the `_INETAF` structure, but somehow a `_LOCAL_ADDRESS` structure is in its place?

That means either `self.InetAF` isn't what we'd expect (but all the JSON suggests it should be) or the dereference in some way changes the type (which it shouldn't do)?

I'm not really seeing how that could happen though...  It wouldn't relate to the image, but to the JSON files instead?  Either the attribute `InetAF` wouldn't look up, or it should return the type from the JSON file (which from all the versions I can see would be an `_INETAF` structure?

I dunno if that helps figure anything out, but just to check, @fgomulka, can you please verify the git commit and whether any additional changes have been made to the code from the main repo develop branch?

**Comment by fgomulka** [comment]
It is git commit [7f6378fd5def39d4e8326edfcc5142f558778fcc](https://github.com/volatilityfoundation/volatility3/commit/7f6378fd5def39d4e8326edfcc5142f558778fcc). And no, no additional changes have been made to the code on `develop`. Let me know if you have any more questions

**Comment by japhlange** [comment]
@ikelos Has there been any change in the last ~4 months regarding how ISFs are parsed or how objects are created with `context.object()`?

The good part is that I can produce it locally. I still don't understand why the object attributes get mixed up though.

**Comment by ikelos** [comment]
The intermed symbols handler was last updated 6 months ago and `context.object` hasn't changed in 2 years.  If you're able to reproduce it locally, is it in an image you can share (or a public one)?  Have you been able to identify whether the structure's members are wrong, or whether the structure itself is the wrong one?  I'm really curious as to how this could be happening...

**Comment by japhlange** [comment]
I can reproduce it by running the plugin but not really in volshell unfortunately.

I can share it, it's just a dev memdump I created for netscan development, so it's a fresh machine. Any idea how to share a 1GB file best?

**Comment by ikelos** [comment]
Google drive is how I'd typically do it (if you have the space)?  mike.auty@gmail.com if you want to share it directly...

**Comment by ikelos** [comment]
It looks as though this should have been resolved due to pull request #536 having been merged.  I'm gonna close this, but if you're still encountering the issue, please let us know...
